llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -1191,6 +1191,11 @@ class VPInstruction : public VPRecipeWithIRFlags {
     SLPStore,
     ActiveLaneMask,
     ExplicitVectorLength,
+    /// Creates a scalar phi in a leaf VPBB with a single predecessor in VPlan.
+    /// The first operand is the incoming value from the predecessor in VPlan,
+    /// the second operand is the incoming value for all other predecessors
+    /// (which are currently not modeled in VPlan).
+    ResumePhi,
```

Comment:
Thanks for renaming, worth updating the title and summary of the patch.

Reply:
Title + description should be updated now